### DISCUSSION TITLE
<title> 테스트 용도로 작성!

### DISCUSSION THREAD

**Discussion by yoonjong-park** [community/community#46126] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

오... 이런 게 있었구나..

**Comment by ettaboyle** [comment, marked as answer]
Hi [yoonjong-park](https://github.com/yoonjong-park), thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

I manually translated this post and saw that it was shared for testing purposes. Please know that the GitHub Community on Discussions is a shared public space and we work hard to keep the space full of helpful content. If you are new to Discussions and want to test features and functions, you can do so in your own repos. You can find more information in the [GitHub Docs: GitHub Discussions](https://docs.github.com/en/discussions)

Please do not test Discussions functionality in this repo. Thank you!
